### PR TITLE
Add test your code section to contribution chapter

### PR DESCRIPTION
**Pull request by lennerd**
Add "Test your code" section to contribution chapter of the documentation, to ease contribution for new developers.

I'm not quite sure how to extend the navigation without having a `toctree` on the contribution page itself. Any ideas?

### CODE REVIEW

File: developer/contributing/test-your-code.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Install Composer dependences with ``composer install``.
+
+2. Run ``./bin/jackrabbit.sh``. This will install a local copy of Jackrabbit into the ``./bin`` directory and start the
+   server. If you already have a running server on your machine, you can skip this step. Also if you want the tests to run on
+   different port than the default `8080`, you need to install Jackrabbit on your own.
+
+3. Run the tests with ``./bin/runtests.sh``. This command has two possible options:
+
+   * ``-i``: Initialize the database and run all tests.
```
Comment (danrot) [inline developer/contributing/test-your-code.rst]: `-i` just stands for initialization, you still have to either use the `-a` flag for running all tests or `-t` for specifying a single suite.

**Comment by lennerd** [comment]
Added the `-a` option. For which branch should I make a PR? `develop` or `master`?

File: developer/contributing/pull-requests.rst [hunk 2]
```diff
@@ -3,17 +3,21 @@ Creating a Pull Request
...
 
-* Use a meaningfull name for the pull request.
-* Create the pull request as soon as possible.  
-* Name the branch after the following format: ``<type>/<description>``, where
+* Clone the Sulu repository from git.
+* Add a branch and name it after the following format: ``<type>/<description>``, where
   type is one of ``feature``, ``bugfix``, ``hotfix`` or ``enhancement``. For
   example: ``feature/what-my-pr-does``. Note that dashes should be used
   instead of spaces (not underscores).
+* Add your feature/bugfix/enhancment.
```
Comment (danrot) [inline developer/contributing/pull-requests.rst]: `enhancement`

**Comment by danrot** [comment]
Sry, just encountered another small typo. This branch can be set against master, since this is not a new functionality.

**Comment by lennerd** [comment]
Fixed typo. Branch seems to be up-to-date. Just tell me if you want me to rebase to master.

**Comment by danrot** [comment]
Everything's fine, thank you :smiley:

**Pull request merged**